### PR TITLE
Update Explorer plugin

### PR DESCRIPTION
**Pull request by oviradoi**
Updated the Explorer plugin for x86
Added the Explorer plugin for x64

### CODE REVIEW

**Comment by chcg** [comment]
@oviradoi Could you also add the your repo to the homepage tag. @kobilutil Is it ok for you to use the forked repo from @oviradoi  as main source for future work/issue tracking?

**Comment by oviradoi** [comment]
@chcg I'm not sure what you mean by adding my repo to the homepage tag.

**Comment by oviradoi** [comment]
Sorry, I did not mean to close this pull request.

File: src/pl.x64.json [hunk 1]
```diff
@@ -173,6 +173,16 @@
...
 		},
+		{
+			"folder-name": "Explorer",
+			"display-name": "Explorer",
+			"version": "1.8.5.0",
+			"id": "bd32d2d9da15b5da07197d2e25b641437fac962e0e94f008f761004f98ecc004",
+			"repository": "https://github.com/oviradoi/npp-explorer-plugin/releases/download/v1.8.5/Explorer_x64.zip",
+			"description": "Explorer plugin is a file browser. You can open whatever you want from it in Notepad++, just double click!",
+			"author": "Jens Lorenz",
+			"homepage": ""
```
Comment (chcg) [inline src/pl.x64.json]: @oviradoi I meant adding the github repo here and also in x86 json file

**Comment by ArkadiuszMichalski** [comment]
@oviradoi Can you open bug section in your "Explorer plugin" repo or provide an email address for contact (now I don't see anny possibility through Github so I had to write here).

**Comment by oviradoi** [comment]
@ArkadiuszMichalski I have opened the issues section in the repo

**Pull request merged**